README.md

```diff
@@ -1,32 +1,167 @@
+wins. In mathematical terms, the spoiler effect is when a voting method exhibits failure of a property known as 
+independence of irrelevant alternatives."](https://electionscience.org/library/the-spoiler-effect/)
+
+Check out [this video](https://www.youtube.com/watch?v=JtKAScORevQ) for a more in depth explanation. 
+
+Below is a nice example to read through: 
+
+<p align="center">
+<img src="./img/irv_walkthrough.png" width="420" height="380"/>
+</p>
+
+Even though the Spoiler Effect can happen in almost any election system, [proponents of Ranked Choice Voting say](http://archive3.fairvote.org/reforms/instant-runoff-voting/irv-and-the-status-quo/spoiler-effect/) that it is 
+so uncommon a phenomenon that voters shouldn't have to worry about it. However, a good number of people [say the 
+opposite.](https://psephomancy.medium.com/ranked-choice-voting-doesnt-fix-the-spoiler-effect-80ed58bff72b) So, 
+what's really going on? *That's the question this project aims to answer...*
+
+tktk -- diff btwn exhausted ballot & spoiled effect
 
 ## Methods
-### Obstacles & how we dealt with them:
-- Exhausted ballots: https://democracyjournal.org/arguments/ranked-choice-voting-is-not-the-solution/
-- Noise
-- Probabilities of vote spread across possible candidates
+### Data & Features
+Finding data with which to train our model with was *difficult*. There are hardly any public datasets that include 
+1+ spoiled elections (in an RCV system), let alone tens of thousands. For this reason, we chose one of the most famous 
+spoiled, RCV elections off of which we could model synthetic data: [Burlington's 2009 mayoral election.](http://bolson.org/~bolson/2009/20090303_burlington_vt_mayor.html) In this election, a candidate named Kurt Wright acted as a Spoiler. 
+Without his presence in the candidate pool, Andy Montroll would have won. Due to this election's mishaps, Burlington 
+voted to repeal RCV voting [in 2010.](https://archive.vpr.org/vpr-news/burlington-voters-repeal-instant-runoff-voting/)
 
-## How to run
+After inspecting the [normalized](https://github.com/ranked-vote/rcv-data-format/blob/master/README.md#normalized-vs-raw) data from this election, we concluded that it took the following features to make an "election":
+- A certain amount of noise across the ballots
+- A specific number of candidates
+- A likely distribution of votes
+- A specific number of ballots
 
-## What to expect
+**Noise**
 
-## Available data for public use
+To model the 'noise' that one would expect to see in real-world elections, we created elections with a certain 
+amount of noise. We defined this noise as a random number between 1-15 that would represent the percentage of total 
+ballots 
+in a specific election that would be missing a vote for a certain candidate. For example, if there were 4 possible 
+candidates for an election, a 'noisy' ballot would be one that had votes for 2/4 candidates, instead of 4/4.
```

Comment:
Maybe, we should mention somewhere that it is a partial vote.

Reply:
partial and/or improper 

ex. partial meaning they did not vote for every round while improper could be they voted all 1's or something for all candidates (did not rank)

Reply:
But we are not creating improper votes in our model, are we? Those that are all the same rank..?

Reply:
Yeah I don't think we're creating "improper" ballot (yet) @Anxhela21

Reply:
I'll add the partial stuff!